src/test/java/org/openmrs/contrib/qaframework/automation/ConditionsSteps.java

```diff
@@ -1,79 +1,61 @@
 package org.openmrs.contrib.qaframework.automation;
 
-import io.cucumber.java.After;
-import io.cucumber.java.Before;
-import io.cucumber.java.en.And;
-import io.cucumber.java.en.Given;
-import io.cucumber.java.en.Then;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
 import org.apache.commons.lang3.StringUtils;
 import org.openmrs.contrib.qaframework.RunTest;
 import org.openmrs.reference.page.ConditionPage;
 import org.openmrs.reference.page.ConditionsPage;
 import org.openqa.selenium.By;
 
-import static org.junit.Assert.*;
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
```

Comment:
Hello my dearest @insookwa , Any progress on this ticket

Reply:
Hahahahha. Most affectionate @sherrif10 😊